pages/common/rbt.md

```diff
@@ -0,0 +1,24 @@
+# rbt
+
+> RBTools is a set of command line tools for working with Review Board and RBCommons.
+> Homepage: <https://www.reviewboard.org/docs/rbtools/dev/>.
+
+- Post changes to Review Board:
```

Comment:
Should this be ....`to a review board`? Or is `review board` the name of the software itself?